### ISSUE TITLE
Fail if duplicate class

### ISSUE THREAD

**Issue by Santana1053** [schrapel/carbon-fields-hidden#1]
It is always best practice to wrap your class around `if ( ! class_exists( 'Hidden_Field' ) )` in case multiple plugins/theme use it.

**Comment by schrapel** [comment]
Definitely. I just took the carbon fields template and put this together
quickly for a project. I'll accept a PR
